src/site/content/en/lighthouse-best-practices/doctype/index.md

```diff
@@ -1,32 +1,43 @@
 ---
 layout: post
-title: Document missing doctype
+title: Page lacks the HTML doctype, thus triggering quirks mode
 description: |
-  Learn about `doctype` audit.
+  Learn how to make sure your page doesn't trigger quirks mode in older
+  browsers.
 web_lighthouse:
   - doctype
+updated: 2019-08-28
 ---
 
 Specifying a doctype prevents the browser from switching to
-[quirks-mode](https://developer.mozilla.org/en-US/docs/Web/HTML/Quirks_Mode_and_Standards_Mode).
-Lighthouse flags your page when it's missing `<!DOCTYPE html>`:
+[quirks mode](https://developer.mozilla.org/en-US/docs/Web/HTML/Quirks_Mode_and_Standards_Mode),
```

Comment:
It's formatted as `quirks-mode` in the Lighthouse UI: https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/dobetterweb/doctype.js#L15

But I suppose in this case that `quirks mode` makes more grammatical sense than `quirks-mode`... the hyphen makes it seem like `quirks-mode` is a compound adjective

Reply:
Yeah, that's an error in the Lighthouse audit title. I pinged issue #1384 in which we discussed updating the LH audit titles and asked whether that's something I can tackle since you're OOO for another few days.

Reply:
Yes please do tackle

<sup>preferably like a linebacker</sup>